src/Sarif.Driver/OrderedFileSpecifier.cs

```diff
@@ -47,11 +47,6 @@ private IEnumerable<IEnumeratedArtifact> EnumeratedArtifacts()
         {
             string normalizedSpecifier = Environment.ExpandEnvironmentVariables(this.specifier);
 
-            if (Uri.TryCreate(this.specifier, UriKind.RelativeOrAbsolute, out Uri uri))
```

Comment:
>e [](http://example.com/codeflow?start=25&length=1)

I would expect a unit test in `OrderedFileSpecifierTests` for a change like this.

Reply:
There is no such suite AFAICT.  After investigating this more, I think all our coverage of this class comes as pass-through coverage via UTs that use `MultiThreadedAnalyzeCommandBase`.   `ValidateCommandTests` does.  Possibly others.  I'll make sure it's covered by the next iteration via such a mechanism, at least.  If you want to see a new suite for `OrderedFileSpecifierTests` in that iteration, just yell and I'll make sure to get that in.